examples/nlp/gpt/conf/gpt_ppo_actor.yaml

```diff
@@ -22,17 +24,44 @@ trainer:
     gae_lambda: 0.95
     normalize_advantages: True
 
-    # pick up from the model
-    # *do not change this*
-    model_gbs: ${model.global_batch_size}
-    model_mbs: ${model.micro_batch_size}
 
-    # the sequence length to pad the rollout batch to
-    # this reduces fragmentation at the cost of using more
+    # the sequence length to pad the rollout batch for training to
+    # reduce fragmentation at the cost of using more
     # memory, set to null if we don't want to pad it
     # to a constant size
+    # if actual seq length is higher than this a warning will be raised
+    # but will not crash and training will still proceed on the larger
+    # sequence length
     rollout_batch_seq_length: null
 
+    # Accelerate training times by accelerating inference stage using TRTLLM
+    trt_llm:
+      enable: False
+      reshard: False # if True then reshard the model into TP only for inference
```

Comment:
it's probably better to keep this as False, mostly because this is a complicated feature. We do tell users in the CHANGELOG to set it to True though(but I think the wording is as follows):
```
- Added TRT-LLM support in PPO. This can be enabled by `trainer.ppo.trt_llm.enable=True`. There is also a reshard option to reshard out pipeline parallelism during inference (i.e running tensor and data parallel only) for further speedup via `trainer.ppo.trt_llm.reshard=True`.
```

think that's ok? reading it again it feels like a opt-in rather than it should be always True

Reply:
Works for me but if it works out-of-the-box and is faster why not enable it by default? (users don't need to know that the implementation is complicated)

Reply:
reshard is a relatively new feature and we've tested it on nemotron/llama but i'm not confident that it works for all possible models, so I'm scared to turn it on by default. also on large models having this to be True will OOM since we can't reshard down to a particular PP size, only PP =1 atm

Reply:
Got it, makes sense!